eg/led-PCA9685.js

```diff
@@ -0,0 +1,24 @@
+var five = require("../lib/johnny-five.js");
+
+five.Board().on("ready", function() {
+
+  var ledPin = process.argv[2] || 0;
+  var led = new five.Led({
+    address: 0x40,
+    pin: ledPin,
+    controller: "PCA9685"
+  });
+
+  // address: The address of the shield.
```

Comment:
Since `0x40` is the default, you can remove the property from this constructor opts